### ISSUE TITLE
string pattern not supported in openapi 3

### ISSUE THREAD

**Issue by aaroncouture** [interagent/committee#222]
I am attempting to use Committee::Middleware::RequestValidation to validate against an OpenApi v3 schema:
```yaml
openapi: 3.0.0
paths:

  /api/v1/events/{id}/bookable_spaces:
    get:
      description: get bookable spaces for an event
      parameters:
      - name: start_time
        in: query
        description: starting time for space availability check
        required: true
        schema:
          type: string
          pattern: '[0-9]{1,2}:[0-9]{2}'
      - name: end_time
        in: query
        description: ending time for space availability check
        required: true
        schema:
          type: string
          pattern: '[0-9]{1,2}:[0-9]{2}'
```

Unfortunately the `pattern` option does not appear to be honored. Is this a known issue or am I missing something?

**Comment by ota42y** [comment]
Sorry, openapi_parser 0.2.4 don't support pattern option yet.

I released 0.2.5 now, so please use this version.
https://rubygems.org/gems/openapi_parser/versions/0.2.5

If you have any problems please reopen this issue

**Comment by aaroncouture** [comment]
Just tested using the new openapi_parser version and it’s working as expected
